packages/network/src/api/image/requests.ts

```diff
@@ -323,6 +332,12 @@ function createLocalImage(options: AddImageOptions): Image {
   return image;
 }
 
+function fetchThumbnail(options: AddImageOptions, image: Image): void {
+  if (options.uploadType !== ImageUploadType.VIDEO_FRAME && options.enableThumbnail) {
+    ky.get(image.thumbnailPath);
+  }
+}
+
```

Comment:
Rename function to `precacheThumbnail`. Also, this funciton should return a promise like this :

```javascript
return options.uploadType !== ImageUploadType.VIDEO_FRAME && options.enableThumbnail
  ? ky.get(image.thumbnailPath).then(() => {})
  : Promise.resolve();
```